rlimit_posix.go

```diff
@@ -4,20 +4,26 @@ package caddy
 
 import (
 	"fmt"
+	"os"
 	"syscall"
 )
 
-// checkFdlimit issues a warning if the OS limit for
-// max file descriptors is below a recommended minimum.
-func checkFdlimit() {
-	const min = 8192
-
-	// Warn if ulimit is too low for production sites
+// SetRlimitOpenFiles set rlimit for NOFILE
+// setcap cap_sys_resource+ep for the caddy binary is needed
+// if it is executed as non-root user
+func SetRlimitOpenFiles(limit uint64) {
```

Comment:
Good question. In my opinion the user should be aware of what he is doing. If someone wants to do that, he should do it. I could add a warning in this case.